networks/ligand_networks_for_developers.ipynb

```diff
@@ -138,17 +137,16 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "from openfe.setup import LigandAtomMapper\n",
+    "from openfe.setup.atom_mapping import LigandAtomMapper\n",
     "\n",
     "class CustomMapper(LigandAtomMapper):\n",
     "    def __init__(self, remove_middle):\n",
     "        super().__init__()\n",
     "        self.remove_middle = remove_middle\n",
     "    \n",
     "    def _mappings_generator(self, molA, molB):\n",
```

Comment:
Do we want to rename things to `componentA/B` or is it suitably clear if we keep `molA/B`?

Reply:
I think it's probably better of two evils to use componentA/B consistently and explain why.

Reply:
Done! please revisit. I didn't completely excise the word "molecule," but I used component where I thought it made sense, and added a bit about components.